app/assets/stylesheets/views/_dashboard.scss

```diff
@@ -0,0 +1,50 @@
+// Dashboard
+
+.assignment_summary {
+  background-color: $light-grey;
+  border-radius: $radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, display, margin, min-width, padding, text-align, width